src/main/connector/system/WindowsProcess/WindowsProcess.yaml

```diff
@@ -0,0 +1,119 @@
+extends:
+- ../System/System
+connector:
+  displayName: WindowsProcess
+  platforms: Any platform with Windows
+  reliesOn: WMI
+  information: Gives OS specific information and metrics
+  detection:
+    connectionTypes:
+    - remote
+    - local
+    appliesTo:
+    - nt
+    criteria:
+    - type: wmi
+      namespace: root\CIMv2
+      query: SELECT * FROM Win32_OperatingSystem
+    tags: [ system, windows ]
+monitors:
+  process:
+    simple:
+      sources:
+        processQuery:
+          # 1 ProcessId;LifeTime;Name;ParentProcessId;ExecutablePath;
+          # 6 ThreadCount;PageFaults;PageFileUsage;ReadOperationCount;ReadTransferCount;
+          # 11 WriteOperationCount;WriteTransferCount
+          type: wmi
+          namespace: root\CIMv2
+          query: | 
+            SELECT
+            ProcessId,
+            CreationDate,
+            Name,
+            ParentProcessId,
+            ExecutablePath,
+            ThreadCount,
+            PageFaults,
+            PageFileUsage,
+            ReadOperationCount,
+            ReadTransferCount,
+            WriteOperationCount,
+            WriteTransferCount
+            FROM Win32_Process
+        sqlQuery:
+          type: sql
+          tables:
+          - source: ${source::processQuery}
+            alias: processQuery
+            columns:
+            - name: ProcessId
+              type: INT
+              number: 1
+            - name: CreationDate
+              type: VARCHAR(30)
+              number: 2
+            - name: Name
+              type: VARCHAR(255)
+              number: 3
+            - name: ParentProcessId
+              type: INT
+              number: 4
+            - name: ExecutablePath
+              type: VARCHAR(255)
+              number: 5
+            - name: ThreadCount
+              type: BIGINT
+              number: 6
+            - name: PageFaults
+              type: BIGINT
+              number: 7
+            - name: PageFileUsage
+              type: BIGINT
+              number: 8
+            - name: ReadOperationCount
+              type: BIGINT
+              number: 9
+            - name: ReadTransferCount
+              type: BIGINT
+              number: 10
+            - name: WriteOperationCount
+              type: BIGINT
+              number: 11
+            - name: WriteTransferCount
+              type: BIGINT
+              number: 12
+          query: |

```

Comment:
It would be helpful to add a comment here explaining how the `${var::processName}` variable should be configured, and including some examples would be beneficial.